multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/ApplicationStager.java

```diff
@@ -36,8 +36,8 @@ public StagingState getStagingState() {
         UUID buildGuid = context.getVariable(Variables.BUILD_GUID);
         if (buildGuid == null) {
             return ImmutableStagingState.builder()
-                                        .state(PackageState.STAGED)
-                                        .build();
+                    .state(PackageState.STAGED)
```

Comment:
It looks like is not applied right formatter

Reply:
Fixed